src/parser.cpp

```diff
@@ -126,13 +126,13 @@ struct Parser::ParserImpl
      * components or units to the @p model. If any attributes exist in @p import source
      * with names matching those in @p node, they will be overwritten. Likewise,
      * any imported components or units of the same name already in @p model will
-     * be overwritten by those parsed from @p node.
+     * be overwritten by those parsed from @p node.cmake
```

Comment:
`cmake` ???

Reply:
Ooops! Typed in the wrong window I guess ... 6e455f8c